roles/shared_storage/tasks/main.yml

```diff
@@ -22,7 +22,7 @@
 
```

Comment:
Maybe add some comment what this role does.

Reply:
It implements some logic with regards to what to mount where that is specific to how we @ UMCG try to use best practices for data management. This is described up to a certain extent in both the end user documentation and in the technical design for the cluster, but I understand some explanation here would be nice. That is the same for all other roles though; maybe we need a documentathon...

Reply:
For the record: I started a README in this roles subdir, but I'm not sure if we want to wait with deploying ```prm03``` on Gearshift for the documentation to be complete...

Reply:
After thinking this over, i must admit my issue is not so much with the implementation of the shared_storage role but whit its design:

I object against putting all the groups in the playbook. In my opinion this should have been handled in the filesystem. Now we have to change the playbook each time a group is added or removed.

Having said this, I'm willing to merge this. Just don't call me when there are issues with it.

Reply:
Groups are handled by but not defined in the playbook. Groups are defined using variables in the ```group_vars``` of a cluster. It does mean though that we need to run the playbook or at least the shared_storage role when groups are added/removed.

Having complete file systems mounted 'as is' may be convenient for sys admins as at aligns with their perspective/view of the system, but does not match with the end user's perspective of the system. It also does not help with providing stable paths to data for good data management independent of which physical file system the data happens to be stored in at some point of time, which may change when data migrated from one system to another. Having said that, if there is an easier way to accomplish the same I'd be happy to hear about it.